src/bioregistry/data/bioregistry.json

```diff
@@ -10286,6 +10286,45 @@
       }
     }
   },
+  "biotop": {
+    "contact": {
+      "email": "stefan.schulz@medunigraz.at",
+      "github": "steschu63",
+      "name": "Stefan Schulz ",
```

Comment:
space in name, should add a more generic test against leading/trailing whitespaces